### ISSUE TITLE
configure dist lib name (again)

### ISSUE THREAD

**Issue by nebiun** [dvidelabs/flatcc#108]
In 0.5.3 is missing a part of the solution:

```
diff src/runtime/CMakeLists.txt ../original/flatcc-0.5.3/src/runtime/CMakeLists.txt
15c15
<     install(TARGETS flatccrt DESTINATION ${lib_dir})
---
>     install(TARGETS flatccrt DESTINATION lib)
```
Without this, libraries are generated in lib64 but installed in lib:

-- Installing: /tmp/target/usr/lib/libflatccrt_d.so
-- Installing: /tmp/target/usr/lib/libflatcc_d.so

**Comment by nebiun** [comment]
And this, too:

```
 diff src/compiler/CMakeLists.txt ../original/flatcc-0.5.3/src/compiler/CMakeLists.txt
41c41
<     install(TARGETS flatcc DESTINATION ${lib_dir})
---
>     install(TARGElatcc DESTINATION lib)
```

And we have:

-- Installing: /tmp/target/usr/lib64/libflatccrt_d.so
-- Installing: /tmp/target/usr/lib64/libflatcc_d.so

**Comment by nebiun** [comment]
Sorry, I have same problem with comment...

**Comment by mikkelfj** [comment]
can you make a pull request?

**Comment by nebiun** [comment]
How I can do it?
I'm not so friendly with github: I've cloned master repository, do the changes but how I can create a pull request? I need a fork of the project ? Or I can use an existent branch? Or I must create a new branch?
I need help :(

**Comment by mikkelfj** [comment]
I'm not an export, but I believe you need to make a fork with a new branch. This will be done automatically by pressing the new pull request button at the front page next to the branch selector.

**Comment by nebiun** [comment]
Done.
I hope :)

**Comment by mikkelfj** [comment]
Seems to work, although you are working on your own master branch - often people use a separate branch name to distinguish between different pull requests (PR's).

**Comment by mikkelfj** [comment]
Fixed in #109
